Code/Tools/ProjectManager/Source/GemCatalog/GemCatalogHeaderWidget.cpp

```diff
@@ -288,29 +289,40 @@ namespace O3DE::ProjectManager
         }
     }
 
-    void CartOverlayWidget::GemDownloadProgress(const QString& gemName, int percentage)
+    void CartOverlayWidget::GemDownloadProgress(const QString& gemName, int bytesDownloaded, int totalBytes)
     {
         QWidget* gemToUpdate = m_downloadingListWidget->findChild<QWidget*>(gemName);
         if (gemToUpdate)
         {
             QLabel* progressLabel = gemToUpdate->findChild<QLabel*>("DownloadProgressLabel");
-            if (progressLabel)
+            QProgressBar* progressBar = gemToUpdate->findChild<QProgressBar*>("DownloadProgressBar");
+
+            if (totalBytes != 0)
```

Comment:
nit: I would leave a comment here making clear why totalBytes would be 0 so you are handling it this way.